docs/reference/content/driver-reactive/getting-started/quick-start-primer.md

```diff
@@ -49,12 +49,12 @@ where `T` is the type of response for the operation.
 
 {{% note %}}
 All [`Publishers`](http://www.reactive-streams.org/reactive-streams-1.0.1-javadoc/?org/reactivestreams/Publisher.html) returned 
-from the API are cold, meaning that no I/O happens until they are subscribed to and the subscription makes a request. So just creating a 
-`Publisher` won't cause any network IO. It's not until `Subscription.request()` is called that the driver executes the operation.
+from the API are cold, meaning that nothing happens until they are subscribed to. So just creating a
```

Comment:
Would it be helpful to link [cold](https://projectreactor.io/docs/core/release/reference/#reactor.hotCold)?

Reply:
Done